using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -104,39 +104,70 @@ If the provided URL is nil or empty, it is returned as is. Otherwise, the follow
 
 * _baseUrl_ is the URL to which the visitor information needs to be appended. If the visitor information is nil or empty, the URL is returned as is.
 * _callback_ is invoked after the updated URL is available.
-* _completionHandler_ is invoked with _urlWithVersionData_ after the updated URL is available or _error_ if an unexpected exception occurs or the request times out. The default timeout is 500ms.
+* _completionHandler_ is invoked with _urlWithVersionData_ after the updated URL is available or _error_ if an unexpected exception occurs or the request times out. The returned `NSError` contains the [ACPError](../../mobile-core-api-reference#acperror) code of the specific error. The default timeout is 500ms.
 
 **Examples**
 
 **Objective-C**
 
 ```objectivec
-NSURL* url = [[NSURL alloc] initWithString:@"www.myUrl.com"];
+NSURL* url = [[NSURL alloc] initWithString:@"https://example.com"];
 [ACPIdentity appendToUrl:url withCallback:^(NSURL * _Nullable urlWithVisitorData) {    
-// handle the appended url here}
+	// handle the appended url here
+	if (urlWithVisitorData) {
+		// APIs which update the UI must be called from main thread
+		dispatch_async(dispatch_get_main_queue(), ^{
+			[[self webView] loadRequest:[NSURLRequest requestWithURL:urlWithVisitorData]];
+  	}
+  } else {
```

Comment:
Formatting is a bit off here.

Reply:
Typora was using tabs. I replaced them with spaces. Hopefully that should fix it.